fed/mosaic-sumo/src/test/java/org/eclipse/mosaic/fed/sumo/junit/SumoTraciRule.java

```diff
@@ -128,7 +131,9 @@ private void before() throws Throwable {
         // startup sumo
         final List<String> startArgs = Lists.newArrayList("-c", scenarioConfig.getName(),
                 "-v", "--remote-port", Integer.toString(port),
-                "--step-length", String.format(Locale.ENGLISH, "%.2f", (double) sumoConfig.updateInterval / 1000d));
+                "--step-length", String.format(Locale.ENGLISH, "%.2f", (double) sumoConfig.updateInterval / 1000d),
+                "--xml-validation", "never"
```

Comment:
We could use `auto` but then we need to make sure that everyone has set SUMO_HOME, otherwise validation might fail.
We could use `always`, but then we'd need to fix all sumo files for the unit tests. I guess this would not be too bad actually.

Reply:
I just noticed that all files used in integration tests are already compliant, so I will commit `always` here and we'll see what happens.